index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css.css">
+    <title>Maidan</title>
+  </head>
+  <body>
+    <div class="container">
+      <div id="search">
+        <h2>Search request</h2>
+        <textarea id="searchInputField" onkeyup="adjustInputFieldSize()" name="name" rows="1" cols="70"></textarea>
```

Comment:
Hmm, `<textarea>` with `rows=1`, something is not very fun here. Why not `<input type="text">` with static big width?

Reply:
\+1 to this
It should be the input field.
Is this a `tags` field from mockup?
Then it should be pretty big and why it needs `adjustInputFieldSize()`?

Reply:
It is the tags field, and the adjustsize part is probably to dynamically allocate page size for this field, but I do think it could be better solved, perhaps by scrollbars of sorts.